Assignment4/src/middleware/counterMiddleware.js

```diff
@@ -0,0 +1,17 @@
+const counterMiddleware = (limit) => {
```

Comment:
Install lint in your vs code editor.It will let you know what are the old coding conventions and unneccesary lines. eg - 
this - > res.status(429).json({ error: "Too Many Requests" });
      return;
can be - > return res.status(429).json({ error: "Too Many Requests" });

Reply:
I have configure the eslint in my Assignment